src/nvim/decoration.h

```diff
@@ -82,7 +82,11 @@ typedef struct {
   Decoration decor;
   int attr_id;  // cached lookup of decor.hl_id
   bool virt_text_owned;
-  int win_col;
+  /// Screen column to draw the virtual text.
+  /// When -1, the virtual text may be drawn after deciding where.
+  /// When < -1, the virtual text shouldn't currently be drawn.
```

Comment:
"not currently" is not very specific. I assume it means "on a later screenline". Perhaps this could be mentioned in the comment.